cmd/root/auth.go

```diff
@@ -270,3 +280,11 @@ func AccountClient(ctx context.Context) *databricks.AccountClient {
 	}
 	return a
 }
+
+func ConfigUsed(ctx context.Context) *config.Config {
+	cfg, ok := ctx.Value(&configUsed).(*config.Config)
```

Comment:
Can't we use `w.Config` instead of pulling it from the context?

The "live" version will also have other properties filled in that are resolved when authenticating.

Reply:
w.Config does not exist (the whole workspace client does not) when authentication failed.
We use w.Config on success though